.github/workflows/main.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+    push:
+        branches: [ main ]
+        # Release tag format is v[version]
+        # For example: v1.3.5
+        tags: [ 'v*' ]
+    pull_request:
+        branches: [ main ]
+
+jobs:
+    check:
+        name: Check
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout sources
```

Comment:
Is it intended to have three spaces after `-` symbol?

Reply:
studio formats yml files this way

Reply:
It's weird, because it is very different from code style specified in .editorconfig 🤔 
Could you try to reformat this file again after .editorconfig update?